### ISSUE TITLE
apk files cannot be installed on Xiaomi Redmi 1 with version 4.2.1 HM2013022

### ISSUE THREAD

**Issue by truebit** [xiaocong/uiautomator#171]
> $ ls -al /usr/local/lib/python2.7/site-packages/uiautomator/libs
> -rw-r--r--  1 foobar  admin  1239378  7  5 18:10 app-uiautomator-test.apk
> -rw-r--r--  1 foobar  admin  1523739  7  5 18:10 app-uiautomator.apk
> -rw-r--r--  1 foobar  admin   461493  7  5 18:10 bundle.jar
> -rw-r--r--  1 foobar  admin    34104  7  5 18:10 uiautomator-stub.jar
> $ adb install -rt app-uiautomator-test.apk
>    [100%] /data/local/tmp/app-uiautomator-test.apk
> 	 pkg: /data/local/tmp/app-uiautomator-test.apk
command would stuck here.
On device it would display below error: package interprete error
<img width="313" alt="2016-07-06 12 24 19" src="https://cloud.githubusercontent.com/assets/754906/16607535/9b6ede7c-4374-11e6-8c9e-cfaaf7c4878b.png">

`app-uiautomator.apk` has the same problem.

**Comment by truebit** [comment]
in Python, it would throw `RPC server not started!` error.
I think the error should be more precise on installing the apks.

**Comment by hongbinbao** [comment]
@truebit 
what's your SDK level? 4.2.x should be SDK level 17. 

Device with SDK level < 18 should push bundle.jar/uiautomator-stub.jar to device side.

**Comment by truebit** [comment]
yep, these files were pushed to device. but python-uiautomator seems not invoking them

**Comment by hongbinbao** [comment]
@truebit 
May be you have not the latest release of python uiautomator

uiautomator               - Python Wrapper for Android UiAutomator test tool
  INSTALLED: 0.2.6 (latest)

**Comment by truebit** [comment]
Of course I installed the latest one... 
anyway I gave up uiautomator py and wrote a native uiautomator jar to fulfill my requirements.
